index.html

```diff
@@ -15,12 +15,28 @@
     <!-- navbar setup -->
     <header>
       <!-- phone navigation -->
+       <!-- Mobile Menu Toggle -->
+          <section class="menu-mobile showMenu">
+            <button class="close" type="button"><h2>X</h2></button>
+            <ul class="mobile-links">
+              <li class="mobileLink">
+                <a class="mobileLinkText" href="#">Portfolio</a>
+              </li>
+              <li class="mobileLink">
+                <a class="mobileLinkText" href="#about-me">About</a>
+              </li>
+              <li class="mobileLink">
+                <a class="mobileLinkText" href="#social-media">Contact</a>
```

Comment:
- [ ] Update the id for portfolio, because when clicked it doesn't send you to the projects, the about and contact work as expected :+1:

Reply:
Done